go.mod: Update gvisor_vsock_tap dependency

praveenkumar opened the pull request:
Updating to https://github.com/containers/gvisor-tap-vsock/commit/b79db04d368ef24620553026a977dcff5749e2bc which resolve https://github.com/crc-org/crc/issues/3643 issue.


**Fixes:** Issue #3643

cfergeau commented on go.mod:
```diff
@@ -5,13 +5,13 @@ go 1.19
 	github.com/cavaliergopher/grab/v3 v3.0.1
 	github.com/cheggaaa/pb/v3 v3.1.0
-	github.com/containers/gvisor-tap-vsock v0.5.0
+	github.com/containers/gvisor-tap-vsock v0.6.2-0.20230609094134-b79db04d368e
```
I'd prefer to stick to release gvisor-tap-vsock versions.

praveenkumar replied:
@cfergeau once we have the next release cut we will use it but as of now we have a old release.

anjannath replied:
this fixes a critical bug on CRC, it'd be good to vendor a released gvisor-tap-vscok but since we don't have a newer release with the needed changes, we can do that when there's new release for gvisor-tap-vsock

cfergeau replied:
Since we are the upstream for gvisor-tap-vsock, getting a new release is relatively easy... ;)

praveenkumar commented:
/retest

praveenkumar committed:
go.mod: Update gvisor_vsock_tap dependency

Updating to https://github.com/containers/gvisor-tap-vsock/commit/b79db04d368ef24620553026a977dcff5749e2bc
which resolve https://github.com/crc-org/crc/issues/3643 issue.

mmurhamm commented:
@praveenkumar , is there a patch I can use or do I need to wait for the next release of crc? Thanks for fixing this.

anjannath commented:
@mmurhamm `crc-2.21.0` will be released soon (1-2 days) otherwise you can build from current master, following this guide: https://github.com/crc-org/crc/blob/main/developing.adoc
